EveryVoiceTTS/EveryVoice
# feat: converted setup.{py,cfg} into pyproject.toml

### PR Goal? 

Migrate `setup.{py,cfg}` to `pyproject.toml` before `pip` forces use to do so.

### Fixes? 

A `pip` deprecation warning.

### Feedback sought? 

Merge approval

### Priority? 

low

### Tests added? 

none

### How to test? 

1. Create a reference environment with `make-everyvoice-env`.
2. Create a new environment with `make-everyvoice-env`.
3. Compare the content of the two environments `diff <(conda env export --name EveryVoice.main_afca68906) <(conda env export --name EveryVoice.pyproject)`.
4. Possibly, examine the logs created during `make-everyvoice-env` but due to some arbitrary order of the logs, this could be tedious.
5. 5. Double check that installing EveryVoice in editable mode still works.

### Confidence? 

fair

### Version change? 

No

### Related PRs? 

None
